PlayerCollection.cs

```diff
@@ -108,6 +108,8 @@ public class PlayerDisplay
         private DetailsButton _detailsButton;
         private Dropdown _dropdown1;
         private Dropdown _dropdown2;
+        private Image _roleIcon1 = new Image { Size = new Point(32, 32) };
```

Comment:
A dropdown's default height is 27, so the image is now aligned with the bottom of the dropdown!
```suggestion
        private Image _roleIcon1 = new Image { Size = new Point(27, 27) };
```